xml/XSLT/CSVindex.html

```diff
@@ -1,40 +1,56 @@
+    <link rel="icon" href="http://jmri.sourceforge.net/images/jmri.ico" type="image/png">
+    <link rel="home" title="Home" href="/">
+    <!-- /Style -->
 </head>
 <body>
+<div class="nomenu" id="mBody">
+    <a href="/index.html"><img src="http://jmri.sourceforge.net/images/logo-jmri.gif" align="right" width="160" height="135" hspace="0" vspace="0"></a>
+    <div id="mainContent">
+    <h1>JMRI Decoder Comma-Separated Values files</h1>
+    <p>
+    <ul>
+        <li><a href="pages/decodersizes.csv">decodersizes.csv</a>&nbsp;&nbsp;&nbsp;Physical size for each defined decoder</li>
+        <li><a href="pages/decoderpower.csv">decoderpower.csv</a>&nbsp;&nbsp;&nbsp;Power limits and connector by decoder</li>
+        <li><a href="pages/cvsummary.csv">cvsummary.csv</a>&nbsp;&nbsp;&nbsp;CVs present in each decoder -
+        this is limited by the CSV file format, and doesn't include all the information in the JMRI decoder definitions</li>
+        <li><a href="pages/reservedCVs.csv">reservedCVs.csv</a>&nbsp;&nbsp;&nbsp;NMRA reserved CVs present in each decoder</li>
+    </ul>
+        <p>Right-click on one of the underlined hyperlinks and choose "Download linked file" to save a local
+        copy.</p>
+        <p>International users using a comma as the decimal separator may want to replace the comma used in these
+        files as the field separator by a semicolon ";" and then replace the decimal separator period "." by a comma to
+        process these files in their localized spreadsheet application.</p>
 
-<A href="/index.html"><IMG SRC="/images/logo-jmri.gif" ALIGN=RIGHT WIDTH="160" HEIGHT="135" HSPACE="0" VSPACE="0"></A>
+    <!-- footer copied from !include footer, current as of Jan. 2018, JMRI 4.10 -->
+    <hr class="hide">
+    <div id="footer">
+        <ul id="bn">
+            <li>Contact us via the <a href="https://groups.yahoo.com/neo/groups/jmriusers/info">JMRI users</a> Yahoo group</li>
+        </ul>
 
-<h1>JMRI decoder Comma-Separated-Variables files</h1>
-<P>
-<UL>
-<LI><A HREF="pages/decodersizes.csv">decodersizes.csv</a> Physical size for each defined decoder
-<LI><A HREF="pages/decoderpower.csv">decoderpower.csv</a> Power limits and connector by decoder
-<LI><A HREF="pages/cvsummary.csv">cvsummary.csv</a> CVs present in each decoder - this is limited by the CSV format, and 
-doesn't include all the information in the JMRI decoder definitions
-<LI><A HREF="pages/reservedCVs.csv">reservedCVs.csv</a> NMRA reserved CVs present in each decoder
-</UL>
-<A HREF="../../help/en/Acknowledgements.shtml">Many people</a>
-have worked on this, and we thank 
-<A HREF="../../help/en/Acknowledgements.shtml">all of them</a>.
-If you find this information useful, we encourage you to 
-<a href="http://jmri.org/donations.shtml">donate to support JMRI</a>.
+        <p>Thanks and congratulations to
+        <a href="http://jmri.sourceforge.net/help/en/Acknowledgements.shtml" title="Who is JMRI">
```

Comment:
Can you replace the instances of `jmri.sourceforge.net` with `jmri.org`? It is a current implementation detail that they are the same, but if we were ever to have problems with SourceForge's hosting services it would be nice to be able to move to another provider without anything breaking as a result.

Reply:
Done, plus the other occurrences